src/main/java/task/TaskManager.java

```diff
@@ -0,0 +1,96 @@
+package task;
+
+import java.util.ArrayList;
+
+/**
+ * Manages all Task related operations such as updating
+ * the completion status of a task, inserting and deleting
+ * the task from the list, and displaying the task.
+ */
+public class TaskManager {
+
+    /**
+     * Main ArrayList strong the tasks.
+     */
+    private ArrayList<Task> taskArr;
+
+    /**
+     * Initialises the ArrayList.
+     */
+    public TaskManager() {
+        this.taskArr = new ArrayList<>();
+    }
+
+    /**
+     * Marks the task as completed.
+     * @param index
+     */
+    public void checkTask(int index) {
```

Comment:
Check is potentially ambiguous: check the state of a task, or mark the task as done

Reply:
Could check be different verb?

Reply:
thanks for pointing that out!